core/lib/zksync_core/src/state_keeper/io/common.rs

```diff
@@ -67,16 +68,16 @@ pub(crate) fn poll_iters(delay_interval: Duration, max_wait: Duration) -> usize
     ((max_wait_millis + delay_interval_millis - 1) / delay_interval_millis).max(1) as usize
 }
 
-/// Loads the pending L1 block data from the database.
-pub(crate) async fn load_pending_batch(
+pub(crate) async fn load_l1_batch_params(
     storage: &mut StorageProcessor<'_>,
     current_l1_batch_number: L1BatchNumber,
     fee_account: Address,
     validation_computational_gas_limit: u32,
     chain_id: L2ChainId,
-) -> Option<PendingBatchData> {
-    // If pending miniblock doesn't exist, it means that there is no unsynced state (i.e. no transaction
-    // were executed after the last sealed batch).
+) -> Option<(SystemEnv, L1BatchEnv)> {
+    // If miniblock doesn't exist (for instance if it's pending),
```

Comment:
Micronit/bikeshedding: the line splits feel a bit random here (e.g. the lines are unnecessarily short and the length is inconsistent).

Reply:
I guess I'm not a folk, since I read it as "If miniblock doesn't exist (for instance if it's pending)," -> pause (why, I've got no new information yet?) -> "it means that there is no unsynced state" -> pause (I forgot what was on the previous line) -> "(i.e. no transactions were executed after the last sealed batch).". 
Personally I use the following logic: if there is no logical separation between parts of the sentence, do NOT break on commas/periods/etc, break on space to not create a pause; if parts are not related much, do the opposite. E.g. "inject pauses", if that makes sense.

Though not a strong opinion, feel free to keep it as is.